accounts/views.py

```diff
@@ -1069,3 +1066,35 @@ def pending(request):
     }
     tvars.update(paginate(request, pendings, settings.SOUNDS_PENDING_MODERATION_PER_PAGE))
     return render(request, 'accounts/pending.html', tvars)
+
+
+@csrf_exempt
+def donation_complete(request):
+    params = {'cmd': '_notify-validate'}
+    for key, value in request.POST.items():
+        params[key] =  value
+    if settings.DEBUG:
+        paypal_validation_url = "https://www.sandbox.paypal.com/cgi-bin/webscr"
+    else:
+        paypal_validation_url = "https://www.paypal.com/cgi-bin/webscr"
+    req = requests.post(paypal_validation_url, data=params)
+    if req.text == 'VERIFIED':
+        campaign = DonationCampaign.objects.order_by('date_start').last()
```

Comment:
wouldn't it be better to pass the campaign ID to paypal and get it back from them rather than assuming it's the last campaign?